mu4/appshell/dockwindow/dockwindow.cpp

```diff
@@ -322,3 +325,13 @@ void DockWindow::setCurrentPageUri(QString uri)
 
     emit currentPageUriChanged(m_currentPageUri);
 }
+
+QMainWindow* DockWindow::qMainWindow()
```

Comment:
Why not just mainWindow()?

Reply:
At the moment, this is a forced, I hope a temporary method, we need exactly `QMainWindow`.
I do not think that in this case, it is the main window in essence (from an architectural point of view)